janitor/functions/groupby_topk.py

```diff
@@ -41,59 +44,78 @@ def groupby_topk(
         2   22   6   fail
         3   43   2   pass
         4   21   5   fail
-        >>> df.groupby_topk('result', 'age', 3) # Ascending top 3
-        ... # doctest: +NORMALIZE_WHITESPACE
-                  age  id result
-        result
-        fail   4   21   5   fail
-               2   22   6   fail
-        pass   0   20   1   pass
-               1   23   4   pass
-               3   43   2   pass
-        >>> df.groupby_topk('result', 'age', 2, {'ascending':False}) # Descending top 2
-        ... # doctest: +NORMALIZE_WHITESPACE
-                  age  id result
-        result
-        fail   2   22   6   fail
-               4   21   5   fail
-        pass   3   43   2   pass
-               1   23   4   pass
+
+    Ascending top 3:
+
+        >>> df.groupby_topk(by = 'result', column = 'age',  k = 3)
+           age  id result
+        0   20   1   pass
+        1   23   4   pass
+        2   43   2   pass
+        3   21   5   fail
+        4   22   6   fail
+
+    Descending top 2 :
+
+        >>> df.groupby_topk(
+        ...     by = 'result',
+        ...     column = 'age',
+        ...     k = 2,
+        ...     ascending = False,
+        ...     ignore_index = False
+        ...     )
```

Comment:
Same here, + trailing comma on the final kwarg and the indentation of the final parenthesis is wrong.